mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -169,11 +173,10 @@ def ocn_modelvsobs(config, field):
 
         if buildObsClimatologies:
             dsObs = xr.open_mfdataset(obsFileName)
-
-            dsTimeSlice = dsObs.sel(time=slice(timeStart, timeEnd))
-            monthlyClimatology = dsTimeSlice.groupby('time.month').mean('time')
-
-            dsObs = monthlyClimatology.transpose('month', 'lat', 'lon')
+            dsObs.rename({'time': 'Time'}, inplace=True)
+            dsObs = dsObs.transpose('Time', 'lat', 'lon')
```

Comment:
Transpose are notorious for being a bad idea, e.g., they typically indicate things weren't set up correctly and they are expensive to execute.  It doesn't sound like it is a huge problem here but I just wanted to double-check.